hanabi_lib/canonical_encoders.cc

```diff
@@ -576,12 +601,13 @@ int EncodeV0Belief_(const HanabiGame& game,
                     bool shuffle_color,
                     const std::vector<int>& color_permute,
                     std::vector<float>* encoding,
-                    std::vector<int>* ret_card_count) {
+                    std::vector<int>* ret_card_count,
+                    bool using_joint_obs = false) {
```

Comment:
let's avoid having default_arg as much as possible. There will be no backward compatibility issue.

Reply:
Will do.